app/frontend/wallet/shelley/shelley-transaction.ts

```diff
@@ -312,7 +312,12 @@ function ShelleySignedTransactionStructured(
   }
 
   function encodeCBOR(encoder) {
-    return encoder.pushAny([txAux, txWitnesses, txAuxiliaryData])
+    return encoder.pushAny([
+      txAux,
+      txWitnesses,
+      true, // fixed alonzo "isValid" Validation tag
```

Comment:
Just curious - I thought this is not part of the submitted transaction but it's rather computed on-chain after the transaction is submitted - what would be the point in putting here false?

Reply:
```
 the IsValid tag, which indicates whether all scripts that are executed during the script
execution phase validate. The correctness of the tag is verified as part of the ledger rules,
and the block is deemed to be invalid if it is applied incorrectly. If it is set to False, then the
block can be re-applied without script re-validation. This tag cannot be signed, since it is
applied by the block producer.
```